src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -1,35 +1,77 @@
 package codeu.controller;
 
 import java.io.IOException;
+import java.io.StringWriter;
+import java.io.PrintWriter;
 import javax.servlet.RequestDispatcher;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.mockito.ArgumentCaptor;
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.Mockito;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
 
 public class RegisterServletTest {
```

Comment:
Great job writing tests! Does anyone see any additional cases that could be added? (I see one in particular.)

Reply:
I think i missed existing user so i added a method for that

Reply:
Awesome--great work! (Sorry for the cryptic comment--I'd like to see everyone more active in code reviews, but you beat everyone else to the punch!)